pages/dr9/sga.rst

```diff
@@ -10,178 +10,833 @@
+``[G,R,Z]_STOP_CODE``                                      int16[N]                                                  Fitting stop code (see `photutils.isophote.IsophoteList`_ and `photutils.isophote.Isophote`_).
+``[G,R,Z]_NDATA``                                          int16[N]                                                  Number of data points used for the fit (see `photutils.isophote.IsophoteList`_).
+``[G,R,Z]_NFLAG``                                          int16[N]                                                  Number of points rejected during the fit (see `photutils.isophote.IsophoteList`_).
+``[G,R,Z]_NITER``                                          int16[N]                                                  Number of fitting iterations (see `photutils.isophote.IsophoteList`_).
+``[G,R,Z]_COG_SMA``                                        float32[M] pixel                                          Semi-major axis position for the curve-of-growth aperture photometry measurements, where ``M`` is the total number of samples (in `arcsec`) along the semi-major axis.
+``[G,R,Z]_COG_MAG``                                        float32[M] AB mag                                         Aperture photometry within the semi-major axis given by ``[G,R,Z]_COG_SMA``.
+``[G,R,Z]_COG_MAGERR``                                     float32[M] AB mag                                         Uncertainty in ``[G,R,Z]_COG_MAG`` (:math:`1\sigma`).                       
+``[G,R,Z]_COG_PARAMS_MTOT``                                float32    AB mag                                         Best-fitting parameter :math:`m_{1}` based on the fit to the curve of growth (see the `Ellipse-Fitting`_ section).
+``[G,R,Z]_COG_PARAMS_M0``                                  float32    AB mag                                         Best-fitting parameter :math:`m_{0}` based on the fit to the curve of growth (see the `Ellipse-Fitting`_ section).
+``[G,R,Z]_COG_PARAMS_ALPHA1``                              float32                                                   Best-fitting parameter :math:`\alpha_{1}` based on the fit to the curve of growth (see the `Ellipse-Fitting`_ section).
+``[G,R,Z]_COG_PARAMS_ALPHA2``                              float32                                                   Best-fitting parameter :math:`\alpha_{2}` based on the fit to the curve of growth (see the `Ellipse-Fitting`_ section).
+``[G,R,Z]_COG_PARAMS_CHI2``                                float32                                                   Reduced :math:`\chi^{2}` of the fit to the curve of growth. *Note: large values of :math:`\chi^{2}` indicate a poor or problematic fit and should be inspected.*
+``RADIUS_SB[22,22.5,23,23.5,24,24.5,25,25.5,26]``          float32    arcsec                                         Semi-major axis length at the :math:`\mu=22`, 22.5, 23, 23.5, 24, 24.5, 25, 25.5, and 26 :math:`\mathrm{mag}\ \mathrm{arcsec}^{-2}` isophote in the *r*-band (-1 if not measured).
```

Comment:
The `22` in 
```
:math:`\mu=22`
```
needs to be outside of the math environment so that the font is rendered consistently with `22.5, 23 ... etc.`.